text.txt

```diff
@@ -0,0 +1,41 @@
+running develop
```

Comment:
remove this file

Reply:
remove this file.
When merging select option "squash commits", so this file won't stored in git history